src/libs/reportUtils.js

```diff
@@ -102,6 +102,16 @@ function isUserCreatedPolicyRoom(report) {
     return lodashGet(report, ['chatType'], '') === CONST.REPORT.CHAT_TYPE.POLICY_ROOM;
 }
 
+/**
+ * Whether the provided report is a Policy Expense chat.
+ * @param {Object} report
+ * @param {String} report.chatType
+ * @returns {Boolean}
+ */
+function isPolicyExpenseChat(report) {
+    return lodashGet(report, ['chatType'], '') === CONST.REPORT.CHAT_TYPE.POLICY_EXPENSE_CHAT;
```

Comment:
NAB I know this is copy-pasta, but curious why we thought it was better to call this with `isPolicyExpenseChat({chatType})` instead of just `isPolicyExpenseChat(chatType)` or `isPolicyExpenseChat(report.chatType)`. Also the use of `lodashGet` seems to suggest something about the params we are passing is optional. Do we expect for a `report` to not have a `chatType` in some case?

Reply:
I was curious about that too and assumed that that's the case since it was specified in the issue. Maybe @TomatoToaster can confirm it?

Reply:
Ah missed this comment earlier. We do this because in most places this _will be_ used, we're pulling the whole report object so it's often going to be `isPolicyExpenseChat(report)`. For use within the `getChatReportName()` function specifically though, we haven't set fields like chatType on the `fullReport` so we have to pull it out of the reportNVPs.

Technically instead of calling `ReportUtils.isUserCreatedPolicyRoom({chatType})` or `ReportUtils.isPolicyExpenseChat({chatType})` we could just do `chatType === CONST.REPORT.CHAT_TYPE.POLICY_ROOM` or so. I'm not opposed to that but this seems like less words in a line.

Reply:
I don't remember why we use  lodashGet(report, ['chatType'], '') though, maybe the original intention was so if we were calling it on an IOU report in some View and those don't have chatTypes? (I'm not sure if that's true, I think chatType is almost always set).

But since this is NAB, I think we can merge this.

Reply:
yeah it's fine let's merge it.